sql/ddl.sql

```diff
@@ -769,6 +894,17 @@ COMMENT ON COLUMN punkt.registreringtil IS 'Tidspunktet hvor en ny registrering
 COMMENT ON COLUMN punkt.sagseventfraid IS 'Angivelse af den hændelse der har bevirket registrering af et fikspunktsobjekt';
 COMMENT ON COLUMN punkt.sagseventtilid IS 'Angivelse af den hændelse der har bevirket afregistrering af et fikspunktsobjekt';
 
+COMMENT ON TABLE punktsamling IS 'Gruppering af Punkter i samlinger, typisk i forbindelse med overvågning af punkters stabilitet.';
+COMMENT ON COLUMN punktsamling.objektid IS 'Unikt id til identifikation af punktsamlinger';
+COMMENT ON COLUMN punktsamling.registreringfra IS 'Tidspunktet hvor registreringen er foretaget.';
+COMMENT ON COLUMN punktsamling.registreringtil IS 'Tidspunktet hvor en ny registrering er foretaget på objektet, og hvor denne version således ikke længere er den seneste.';
+COMMENT ON COLUMN punktsamling.sagseventfraid IS 'Angivelse af den hændelse der har bevirket registrering af et fikspunktsobjekt';
+COMMENT ON COLUMN punktsamling.sagseventtilid IS 'Angivelse af den hændelse der har bevirket afregistrering af et fikspunktsobjekt';
+COMMENT ON COLUMN punktsamling.jessenpunktid IS 'Punktsamlingens Jessenpunkt';
+COMMENT ON COLUMN punktsamling.jessenkoordinatid IS 'Den fastholdte koordinat for Jessenpunktet';
+COMMENT ON COLUMN punktsamling.navn IS 'Punktsamlingens navn';
+COMMENT ON COLUMN punktsamling.formaal IS 'Formålet med punktsamlingen, eksempelvis til registrering af hensigt om opmålingsfrekvens etc.';
```

Comment:
jeg skal måske have lidt hjælpt til at forstå eksemplet i denne beskrivelse: Siger det, at punktsamlingen er oprettet for at registrere opmålinger, så man kan holde øje med frekvensen af disse?

Reply:
Nej, pointen var at du her fx kan skrive "Til overvågning af stabiliteten af K-01-2345, genopmåles med geometrisk nivellement hvert tredje år" eller noget i den stil. Jeg tror de konkrete eksempler må vi vende tilbage til, her arbejder jeg ud fra en vag beskrivelse af et behov Aslak har bemærket. Præcis hvordan det i praksis skal bruges er jeg ikke helt sikkert på, men det håber jeg bliver mere klart for os som tiden går.